docs/podman.1.md

```diff
@@ -103,24 +103,21 @@ the exit codes follow the `chroot` standard, see below:
 **_125_** if the error is with podman **_itself_**
 
     $ podman run --foo busybox; echo $?
-    # flag provided but not defined: --foo
-      See 'podman run --help'.
+    Error: unknown flag: --foo
       125
 
 **_126_** if executing a **_container command_** and the the **_command_** cannot be invoked
```

Comment:
crud, didn't spot this earlier.  _container command_ here, _contained command_ below.  Can we make them all the same?  I like the contained thought fwiw.